module/zfs/spa_config.c

```diff
@@ -81,6 +78,7 @@ static int zfs_autoimport_disable = B_TRUE;
 void
 spa_config_load(void)
 {
+#ifndef _KERNEL
```

Comment:
I'd need to refresh my memories on how ZFS root mounting now works on FreeBSD, but just looking on this change it seems like we don't need `spa_boot_init()` and its registration for `mountroot` event also.

Reply:
I've removed `spa_boot_init()`.  I don't think it's needed either, but it be great if someone more familiar with how this works on FreeBSD could confirm that.